core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -97,12 +97,19 @@ oppia.controller('FeedbackTab', [
     };
 
     var _isSuggestionOpen = function() {
-      return $scope.activeThread.status === 'open';
+      return ($scope.activeThread.status === 'received' ||
```

Comment:
For clarity, suggest changing the name since this function no longer just checks that the status is `open`.

Reply:
Done